src/asset-management/AssetManagedPair.sol

```diff
@@ -127,12 +127,5 @@ abstract contract AssetManagedPair is Pair, IAssetManagedPair {
 
             IERC20(token1).transferFrom(address(assetManager), address(this), lDelta);
         }
-
-        _update(
-            _totalToken0(),
-            _totalToken1(),
-            reserve0,
-            reserve1
-        );
```

Comment:
This broke when called midswap right? Any consequence/downside of removing this update?

Reply:
didn't break. It was a conscious decision to remove it. See the very first comment I made [here](https://github.com/vexchange/v3-core/pull/96#discussion_r1005089726)